src/components/EntityInfo.vue

```diff
@@ -223,19 +212,9 @@ export default class EntityInfo extends Mixins(CommonMixin, EnumMixin) {
     return ''
   }
 
-  /** True if the entity has the subject status. */
-  private get isGoodStanding (): boolean {
-    return (this.entityStatus === EntityStatus.GOOD_STANDING)
-  }
-
-  /** True if the entity has the subject status. */
-  private get isPendingDissolution (): boolean {
-    return (this.entityStatus === EntityStatus.PENDING_DISSOLUTION)
-  }
-
-  /** True if the entity has the subject status. */
-  private get isNotInCompliance (): boolean {
-    return (this.entityStatus === EntityStatus.NOT_IN_COMPLIANCE)
+  /** Returns whether business is in good standing */
+  private get isInGoodStanding (): boolean {
```

Comment:
This getter is now duplicated (see also App.vue), so I recommend this be moved to the store getters file.